rootfs/requirements.txt

```diff
@@ -15,3 +15,4 @@ pytz==2015.7
 PyYAML==3.11
 requests==2.9.1
 simpleflock==0.0.3
+git+https://github.com/zx2c4/cryptography.git@master#egg=cryptography
```

Comment:
Could we pin this to SHA 5e7e84c instead? Seems safer than "master" which could change.